### PR TITLE
Allow zooming cycle count yrange.

### PR DESCRIPTION
**Pull request by vext01**
Idle cores cause the axis on busy core to be too zoomed out. Add a switch.
OK?

### CODE REVIEW

File: bin/plot_krun_results [hunk 1]
```diff
@@ -1093,6 +1101,17 @@ if __name__ == '__main__':
         print("invalid --export-size argument")
         sys.exit(1)
 
+    # core cycles y-axis range
```
Comment (snim2) [inline bin/plot_krun_results]: I think you could safely kill this comment

**Comment by snim2** [comment]
Generally this is good, but this is a bit out of step with out we deal with `--xlimits`. For `--xlimits` we have an example of the expected input in the CLI parser:

``` python
                        help="Specify X-axis limits as a comma separated pair "
                             "'start,end'. Samples start from 0. e.g. "
                             "'-x 100,130' will show samples in the range "
                             "100 to 130.")
```

and we don't check that the user has passed in something sensible with a try/except, but we probably should!

**Comment by vext01** [comment]
Will fix the usage, but the xlimit bug is for another day.

**Comment by vext01** [comment]
Try that. Fixed a bug too.

**Comment by snim2** [comment]
I haven't tested with a huge number of options, but this works for me. Want to squash?

**Comment by vext01** [comment]
splat

**Review by snim2 — APPROVED** [review]

**Pull request merged**